infrastructure/Dockerfile

```diff
@@ -42,4 +44,7 @@ ENV JAVA_OPTS -verbose:gc \
 CMD java -cp *.jar \
     ${JAVA_OPTS} \
     -Dlogback.configurationFile=/usr/share/corfu/conf/logback.prod.xml \
-    org.corfudb.infrastructure.CorfuServer
+    org.corfudb.infrastructure.CorfuServer \
+    --compactor-script /usr/share/corfu/scripts/compactor_runner.py \
+    --compactor-config=/usr/share/corfu/conf/corfu-compactor-config.yml \
+    --compaction-trigger-freq-ms=900000
```

Comment:
I think that, currently, this will not enable compaction. CMD is the default command and is executed if in the helm chart the command is not specified. But we do have the command specified in the helm chart because we made it configurable:
https://github.com/CorfuDB/corfudb-cloud/blob/master/cloud/corfu/corfu/templates/statefulset.yaml#L34

If we want to follow the helm route and make compaction configurable, we can follow the principles outlined in the helm command. For example, we can include `enableCompaction: true` in values.yaml:
https://github.com/CorfuDB/corfudb-cloud/blob/master/cloud/corfu/corfu/values.yaml
And then after this line include https://github.com/CorfuDB/corfudb-cloud/blob/master/cloud/corfu/corfu/templates/statefulset.yaml#L61 something like:
```
{{- if .Values.enableCompaction }}
    --compactor-script /usr/share/corfu/scripts/compactor_runner.py \
    --compactor-config=/usr/share/corfu/conf/corfu-compactor-config.yml \
    --compaction-trigger-freq-ms=900000 \
{{- end }}
```

Reply:
yes the corfudb-cloud change will follow soon - this is just to complete the github corfudb repo side of changes